src/applications/ivc-champva/shared/components/fileUploads/FileUpload.jsx

```diff
@@ -63,8 +63,13 @@ export function FileFieldCustom(props) {
       if (uploadsMissing(props.contentAfterButtons.props.form.data)) {
         props.goToPath('/supporting-files');
       } else {
-        // Since we just uploaded the last file, bypass files overview page
-        props.goToPath('/review-and-submit');
+        // Since we just uploaded the last file, bypass files overview page.
+        // Optionally, jump to a predefined path. Otherwise, go to review page.
+        props.goToPath(
+          props.uploadsCompletePath
+            ? `/${props.uploadsCompletePath}`
+            : '/review-and-submit',
+        );
```

Comment:
This is the main point of the PR. Rather than _always_ navigating to `review-and-submit`, we now check if the optional `uploadsCompletePath` is present and use that instead.